hydra-node/hydra-node.cabal

```diff
@@ -111,6 +111,7 @@ library
     , cardano-ledger-core
     , cardano-ledger-core:testlib
     , cardano-ledger-shelley
+    , cardano-protocol-tpraos
```

Comment:
Why is this package needed? TPraos is a protocol not in use anymore and `hydra-node` should not need to be doing anything with Cardano consensus protocols?